examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,137 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import glide.api.GlideClient;
+import glide.api.logging.Logger;
+import glide.api.models.configuration.GlideClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.TimeoutException;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import static glide.api.logging.Logger.Level.ERROR;
+import static glide.api.logging.Logger.Level.INFO;
+import static glide.api.logging.Logger.Level.WARN;
+import static glide.api.logging.Logger.log;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a <code>GlideClient</code> instance.
+     *
+     * This function initializes a <code>GlideClient</code> with the provided list of nodes.
```

Comment:
Where is the provided list of nodes? Shouldn't that be an argument?

Reply:
In Python, it is implemented with a list of node in argument. In Java and Node, user can specify the node in the local variable of `createClient()`.

Reply:
I can modify the comment and add the similar comment used in Node.